### PR TITLE
test: Add a second MSP to fullnet tests (describeMspNet)

### PR DESCRIPTION
**Pull request by pLabarta**
This PR adds a second MSP node to the fullnet testrunner. It also adds node keys and updates constants related to both the first MSP and the new one.

Extraction of common components used by both BspNet and MspNet testrunners will be included in a following PR.

### CODE REVIEW

File: docker/local-dev-full-compose.yml [hunk 1]
```diff
@@ -42,7 +42,35 @@ services:
...
         "--max-storage-capacity=4294967295",
         "--jump-capacity=1073741824",
-        "--name=sh-msp",
+        "--name=sh-msp-1",
+        "--no-hardware-benchmarks",
+        "--unsafe-rpc-external",
+        "--rpc-methods=unsafe",
+        "--port=30350",
+        "--rpc-cors=all",
+        "--node-key=${NODE_KEY}",
```
Comment (ffarall) [inline docker/local-dev-full-compose.yml:51]: Does this mean that they will both have the same `NODE_KEY`? Shouldn't they have different ones?

File: docker/local-dev-full-rocksdb-compose.yml [hunk 2]
```diff
@@ -46,7 +46,37 @@ services:
...
         "--storage-layer=rocks-db",
         "--storage-path=/tmp/msp/${MSP_ID:-default_msp_id}",
-        "--name=sh-msp",
+        "--name=sh-msp-1",
+        "--no-hardware-benchmarks",
+        "--unsafe-rpc-external",
+        "--rpc-methods=unsafe",
+        "--port=30350",
+        "--rpc-cors=all",
+        "--node-key=${NODE_KEY}",
```
Comment (ffarall) [inline docker/local-dev-full-rocksdb-compose.yml:55]: Same here

**Review by ffarall — APPROVED** [review]
Approved with minor comment. Good job!

File: docker/local-dev-full-compose.yml (hunk 1 above)
Comment (pLabarta) [reply docker/local-dev-full-compose.yml:51]: No, on `test/util/fullNet/helpers.ts`, when containers are started, these are passed their respective `NODE_KEY`. Like so:

```
await compose.upOne("sh-msp-1", {
      cwd: cwd,
      configAsString: updatedCompose,
      log: true,
      env: {
        ...process.env,
        NODE_KEY: ShConsts.NODE_INFOS.msp1.nodeKey,
        BSP_IP: bspIp,
        BSP_PEER_ID: bspPeerId,
        MSP_ID: ShConsts.DUMMY_MSP_ID
      }
    });
```

**Pull request merged**